packages/skill-lesson/server/trpc.ts

```diff
@@ -8,9 +8,9 @@
  * @see https://trpc.io/docs/v10/procedures
  */
 
-import {Context} from './context'
 import {initTRPC} from '@trpc/server'
 import superjson from 'superjson'
+import {Context} from './context'
```

Comment:
probably makes sense to move the context and this file to a dedicated `skill-trpc` package?